testinfra/modules/package.py

```diff
@@ -165,7 +165,14 @@ def version(self):
 class RpmPackage(Package):
     @property
     def is_installed(self):
-        return self.run_test("rpm -q %s", self.name).rc == 0
+        result = self.run("rpm -q %s", self.name)
+        if result.succeeded:
+            return True
+
+        if "not installed" in result.stdout or not result.stdout.startswith(self.name):
```

Comment:
We can be pickier here and make the `stdout` match a regex, just to make sure there is actually a package with its version, etc. @philpep  What do you think?